hardhat/hardhat.config.ts

```diff
@@ -67,12 +70,18 @@ const config: HardhatUserConfig = {
         development: {
             url: devaddress,
             accounts:
-            [privateKey],
+            [
+                privateKey,
+                '011967d88c6b79116bb879d4c2bc2c3caa23569edd85dfe0bc596846837bbc8e',
+                '22a316b515a68d4851087571bd5ff051f5ec3f13b28997fb80a8de055052514e'
+            ]
```

Comment:
@diogosmendonca I dont think we need to add these two new private keys here because the first one is already set in the `privateKey` variable in line 39 and the second private key is the visitor node which we need to test by adding as a deployer so that should not be needed either. I've removed these keys and the process still works.

Reply:
@diogosmendonca I've removed the two private keys you added in my last commit. Do take a look and try this out when you are next around. Many thanks.